starknet_py/tests/e2e/conftest.py

```diff
@@ -211,13 +260,21 @@ def new_gateway_account_client(new_address_and_private_key, gateway_client):
     )
 
 
-@pytest.fixture(scope="module", params=["gateway_account_client", "rpc_account_client"])
+@pytest.fixture(
+    scope="module",
+    params=[
+        "gateway_account_client",
+        "new_gateway_account_client",
+        "rpc_account_client",
+    ],
+)
 def account_client(request):
-    # FIXME add rpc client for devnet tests
     return request.getfixturevalue(request.param)
```

Comment:
Ideally this fixture would automatically include or exclude a rpc client based on network. This can be however addressed at some later time, please create an issue about researching a solution for this.